gson/src/test/java/com/google/gson/functional/ArrayTest.java

```diff
@@ -99,9 +100,7 @@ public void testNullsInArrayDeserialization() {
     String json = "[\"foo\",null,\"bar\"]";
     String[] expected = {"foo", null, "bar"};
     String[] target = gson.fromJson(json, expected.getClass());
-    for (int i = 0; i < expected.length; ++i) {
-      assertThat(target[i]).isEqualTo(expected[i]);
-    }
+    assertThat(target).asList().containsAnyIn(expected);
```

Comment:
Should probably be `containsExactly(...).inOrder()` as well.
```suggestion
    assertThat(target).asList().containsExactly(expected).inOrder();
```